build.sh

```diff
@@ -305,7 +305,7 @@ if [[ -z "${DISABLE_JAVA_UDF}" ]]; then
 fi
 
 if [[ -z "${DISABLE_JAVA_CHECK_STYLE}" ]]; then
-    DISABLE_JAVA_CHECK_STYLE='OFF'
+    DISABLE_JAVA_CHECK_STYLE='ON'
```

Comment:
Donot change this.

Reply:
this is just for test, will remove it if all test passed.